target/linux/ramips/dts/mt7621_dlink_dir-882-r1.dts

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621_dlink_dir-8xx-r1.dtsi"
+
+/ {
+	compatible = "dlink,dir-882-r1", "mediatek,mt7621-soc";
+	model = "D-Link DIR-882 R1";
+
+	aliases {
+		led-boot = &led_power_orange;
+		led-failsafe = &led_power_green;
+		led-running = &led_power_green;
+		led-upgrade = &led_net_orange;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_orange: power_orange {
+			label = "dir-882-r1:orange:power";
```

Comment:
I've just pushed a patch to master that removes the model part of these LED labels. With that change, keys and leds are the same as for dir-8xx-a1 as it appears to me. So, the question arises whether we can move more to the DTSI or even create a shared DTSI for dir-8xx-a1 and dir-8xx-r1 ...

Reply:
Nice work (: I adjusted the DTSI/DTS changes to reflect your patch. At this point, the dtsi file between the 8xx-r1 routers and 8xx-a1 routers is related to the partitioning.

I think there are a few options here:
1. Keep things as I pushed them.
2. Make an 8xx-x1 dtsi, have the 8xx-r1.dtsi and 8xx-a1.dtsi import it, and put the partition section there.
3. Only have an 8xx-x1 dtsi and put the partition section in each model's dts file.

What would you like me to do? I would appreciate if somebody throughly review any changes if I refactor the DTSI files. I only have the DIR882-R1 model to test with, and my knowledge of DTSI and DTS files is rudimentary.

Reply:
I created a few dtsi files to reduce duplication. Please let me know what you think.